src/Components/ebikes/EbikeForm.jsx

```diff
@@ -0,0 +1,109 @@
+// YourComponent.js
+import React, { useState } from 'react';
+import { v4 } from 'uuid';
+import { useDispatch, useSelector } from 'react-redux';
+import { postEbike } from '../../redux/ebike/addingNewbike';
+import NavigationPanel from '../NavigationPanel';
+
+function EbikeForm() {
+  const dispatch = useDispatch();
+  const { locations } = useSelector((state) => state.locationSlice);
+  const { ebikeModels } = useSelector((state) => state.ebikeModelSlice);
```

Comment:
- [ ] If I try to press the submit button without moving both the "location" and "model" options, the state won't load the default model and location (Haibike and Berlin) and the submit button won't work:

![image](https://github.com/khanh-devos/capstone-ebike-reservation-frontend/assets/57421823/ab4ab2e3-a2b6-4128-a21c-e85c3be3dd31)

This is because there is not a default state:

![image](https://github.com/khanh-devos/capstone-ebike-reservation-frontend/assets/57421823/fc4a4d52-c6e8-4946-98da-9a570a9104fd)

The solution is to add a default state (Haibike and Berlin) to the local states defined on line 10 and 11.


Kindly, fix this bug so your app works as intended, even if your users want to leave the defaults Berlin and Haibike untouched. 👍